tfx_bsl/beam/run_inference.py

```diff
@@ -71,27 +73,26 @@
                                classification_pb2.Classifications]]
 
 
+class OperationType(object):
+  CLASSIFICATION = 'CLASSIFICATION'
+  REGRESSION = 'REGRESSION'
+  PREDICTION = 'PREDICTION'
+  MULTIHEAD = 'MULTIHEAD'
+
+
 @beam.ptransform_fn
 @beam.typehints.with_input_types(Union[tf.train.Example,
                                        tf.train.SequenceExample])
-@beam.typehints.with_output_types(prediction_log_pb2.PredictionLog)
```

Comment:
Why is this type hint removed? Are we not emitting this type as an output anymore

Reply:
Not always. If we use remote inference, the transform will emit a response body as an output. The response body, in turn, contains a collection of predictions which is a JSON list[1]. That's why I removed the hint.

[1] https://cloud.google.com/ai-platform/prediction/docs/reference/rest/v1/projects/predict#response-body-details

Reply:
Do I understand correctly that depending on usage, we get different output types here?
If yes, can we standardize output?

Reply:
@Ardagan Sorry, I missed your comment. That's no longer true, the output type in all cases is `PredictionLog` now.